### PR TITLE
Update SPS phase-field and engineering scale two way example 

### LINKED ISSUE #85: Update SPS 2-way coupling example between engineering scale and phase-field
## Reason
Engineering-scale example errors out when stainless steel temperature goes out of calibration range, changing initial and boundary conditions in phase-field model to prevent this from happening
Current example uses only 2-particle geometry in the phase-field model, but testing has shown that 4-particle geometry is more representative. Also change expression for electrical conductivity to account for GB and surface enhancement

## Design
Update input files

## Impact
No changes beyond the input files, they are only tested with a `RunApp` test.

### PR DESCRIPTION
**Pull request by laagesen**
Closes #85

### CODE REVIEW

File: examples/sps/multiapp/electrothermal_with_phase_field/twoway_initial_prototype/micro_yttria_thermoelectric_twoway.i [hunk 1]
```diff
@@ -59,60 +59,56 @@ initial_voltage=0.0001
...
+  [phi_IC]
     type = SpecifiedSmoothCircleIC
     variable = phi
-    x_positions = '40 40'
-    y_positions = '0  40'
-    z_positions = '0   0'
-    radii = '20 20'
+    x_positions = '40 40 80 80'
+    y_positions = '0  40 0  40'
+    z_positions = '  0   0   0   0'
```
Comment (cticenhour) [inline examples/sps/multiapp/electrothermal_with_phase_field/twoway_initial_prototype/micro_yttria_thermoelectric_twoway.i]:
```suggestion
    z_positions = '0   0   0   0'
```

**Review by cticenhour — CHANGES_REQUESTED** [review]

**Comment by laagesen** [comment]
Took a little while to get back to this because I was on travel most of this week, but should be good to go now

**Comment by laagesen** [comment]
Please hold off merging for now, may have discovered another issue

**Comment by laagesen** [comment]
Should be good to go now

**Review by cticenhour — APPROVED** [review]

**Pull request merged**